internal/xds/translator/testdata/out/xds-ir/http-route-dynamic-resolver.clusters.yaml

```diff
@@ -9,11 +9,18 @@
         dnsLookupFamily: V4_PREFERRED
         dnsRefreshRate: 30s
         name: httproute/default/httproute-1/rule/0
-  commonLbConfig:
-    localityWeightedLbConfig: {}
+  commonLbConfig: {}
   connectTimeout: 10s
   dnsLookupFamily: V4_PREFERRED
   lbPolicy: CLUSTER_PROVIDED
+  loadBalancingPolicy:
+    policies:
+    - typedExtensionConfig:
+        name: envoy.load_balancing_policies.least_request
+        typedConfig:
+          '@type': type.googleapis.com/envoy.extensions.load_balancing_policies.least_request.v3.LeastRequest
+          localityLbConfig:
+            localityWeightedLbConfig: {}
```

Comment:
Hey @zhaohuabing do you know why dynamic-resolver isn't compatible with `cluster.loadBalancingPolicy`? Looking at git blame I see you added this exclusion - https://github.com/envoyproxy/gateway/blame/main/internal/xds/translator/cluster.go#L341

Reply:
That did it!